homework/insensitive-palindrom/IsPalindrome.cpp

```diff
@@ -0,0 +1,16 @@
+#include <algorithm>
+#include <cwctype>
+
+bool is_palindrome(std::string word) {
+    std::transform(begin(word), end(word), begin(word), [](auto& el) {
+        if (iswalnum(el)) {
+            el = std::tolower(el);
+            return el;
+        } 
+        el = 'A';
+        return el;
+    });
```

Comment:
This is not correct. Just use remove_copy_if or transform and erse -> iterator returned by transform will be last valid, and rest of it you can erase